infra/Pos/Subscription/Dns.hs

```diff
@@ -44,10 +45,15 @@ type KnownRelays = Map NodeId KnownRelay
 activeRelays :: KnownRelays -> [NodeId]
 activeRelays = map fst . filter (relayActive . snd) . M.toList
 
+-- TODO: Use valency and fallbacks
```

Comment:
No, not in this PR. Figured we'd merge this first, at least the .yaml format is then changed (useful for devops). Was actually kind of hoping to talk to you about this because you've already implemented this logic for Kademlia.